noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -22,6 +22,7 @@ global ARGS_LENGTH: u64 = 16;
 
 // docs:start:constants
 // "PER CALL" CONSTANTS
+// If modifiying, update avm_kernel.pil / avm_constants.pil offset values (our pil does not yet support constant evaluation so values are hardcoded)
```

Comment:
bummer

Reply:
ah here! (sorry for the redundant comments). These should be possible to autogen? :)